src/content/champions/yelle-lieder.yaml

```diff
@@ -15,6 +15,24 @@ social:
   github: yellelieder
   linkedin: lieder
 activities:
+  - contributionType: oss contribution
+    linkedGSFProject: Carbon Aware SDK
```

Comment:
I have not seen this project before and I do not think it is directly linked to https://github.com/Green-Software-Foundation/carbon-aware-sdk . Please specify how was this a contribution to the Carbon Aware SDK, or any related pull requests.

Reply:
@adamj89 can you have a look at this, please?